zipkin-lens/src/components/TracePage/SpanDetail/SpanTags.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withStyles } from '@material-ui/styles';
```

Comment:
Why do we prefer HOC API overs styled-component API or hook API?

Reply:
When using the Hooks API, I need to mock makeStyles.
like https://github.com/mui-org/material-ui/issues/15404#issuecomment-497373555
This is why I chose the HOC style...